app/logic/pager.rb

```diff
@@ -0,0 +1,18 @@
+class Pager
```

Comment:
@vlasiak, As I see, a pager is being used only for UI and it represents some attributes of pagination. That's why the suggestion is to move it to view objects layer, as logic layer should contain business rules only.